src/ape/managers/project.py

```diff
@@ -385,6 +386,10 @@ def _project(self) -> ProjectAPI:
 
         return self._cached_projects[self.path.name]
 
+    @property
+    def _in_source_cache_folder(self) -> Path:
+        return self.contracts_folder / ".cache"
```

Comment:
Would you have to do this with any dependencies? Should basically have everything that's needed in the manifest

Might need to test with recursive dependencies eventually

Reply:
Also maybe a better name:
```suggestion
    def _dependency_cache_folder(self) -> Path:
        return self.contracts_folder / ".cache"
```

Reply:
This is a cache for compilers to put in files that they need to use for compiling dependencies. The files have to be in the same path as the `contracts_folder`.

I don't want to confuse it with the dependency cache that is in `.ape/packages`

Reply:
Fair!

Reply:
Solidity uses the manifests that are cached in `.ape/packages` to put them in this cache